xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/Wrappers.java

```diff
@@ -213,8 +275,13 @@ public static XrpCurrencyAmount ofXrp(BigDecimal amount) {
      * @return A {@link BigDecimal} representing this value denominated in whole XRP units.
      */
     public BigDecimal toXrp() {
-      return new BigDecimal(this.value().bigIntegerValue())
```

Comment:
Was this a bug? If `value()` is drops, we should have been scaling by 10^-6, right?

Reply:
No, this was working normally before -- it's just difficult to see the change-set here in GH. Line 224/286 right below is unchanged, so the scaling was happening there (and still is).

Reply:
Resolving this one for now - feel free to re-open if I missed something.